index.html

```diff
@@ -1933,8 +1933,11 @@ <h3>Verification Material</h3>
 public key.
             </p>
             <p class="advisement">
-Note that publicKeyMultibase is not a standard and is therefore subject to change. There might be some cases where <code><b>public</b>KeyMultibase</code> can
-be represented although <code><b>private</b>KeyMultibase</code> cannot.
+Note that the [[?MULTIBASE]] specification is not yet a standard and is 
+subject to change. There might be some data format use cases where 
+<code><b>public</b>KeyMultibase</code> is defined to express keys 
+publicly, but <code><b>private</b>KeyMultibase</code> is not in order 
+to protect against accidentally leaking secret keys.
```

Comment:
As @peacekeeper noted, that second sentence is a bear.  If I understood it correctly, this rephrasing should help others.  If I got it wrong, I guess we'll need to massage it a bit more.

```suggestion
subject to change. There might be some use cases for this data format
where <code><b>public</b>KeyMultibase</code> is defined, to allow for
expression of public keys, but <code><b>private</b>KeyMultibase</code>
is not defined, to protect against accidental leakage of secret keys.
```

Reply:
I agree this makes it easier to understand, thanks!